Pipfile.lock

```diff
@@ -423,10 +437,17 @@
         },
         "werkzeug": {
             "hashes": [
-                "sha256:87ae4e5b5366da2347eb3116c0e6c681a0e939a33b2805e2c0cbd282664932c4",
-                "sha256:a13b74dd3c45f758d4ebdb224be8f1ab8ef58b3c0ffc1783a8c7d9f4f50227e6"
+                "sha256:7280924747b5733b246fe23972186c6b348f9ae29724135a6dfc1e53cea433e7",
+                "sha256:e5f4a1f98b52b18a93da705a7458e55afb26f32bff83ff5d19189f92462d65c4"
+            ],
+            "version": "==0.16.0"
+        },
+        "zipp": {
+            "hashes": [
+                "sha256:f06903e9f1f43b12d371004b4ac7b06ab39a44adc747266928ae6debfa7b3335",
+                "sha256:f6be885bea0a12d71d1204e0bea6f59fe20777d99edb849da1ff945dc51c4e65"
             ],
-            "version": "==0.15.5"
+            "version": "==0.6.0"
```

Comment:
This looks like a downgrade, but it's just imperfection fo the diff visualization. The 0.15 was werkzeug, 0.6 is zipp.